### PR TITLE
Feature/submissions

### PR DESCRIPTION
**Pull request by estyxx**
Add AudienceLevel

### CODE REVIEW

File: backend/tests/conferences/factories.py [hunk 1]
```diff
@@ -119,6 +119,19 @@ def durations(self, create, extracted, **kwargs):
...
+    @factory.post_generation
+    def audience_levels(self, create, extracted, **kwargs):
+
+        if not create:
+            return
+
+        if extracted:
+            for audience_level in extracted:
+                self.audience_levels.add(
+                    AudienceLevel.objects.get_or_create(name=audience_level)[
```
Comment (marcoacierno) [inline backend/tests/conferences/factories.py]: Just need to fix the formatting here

File: backend/tests/submissions/test_send_submission.py [hunk 2]
```diff
@@ -22,13 +22,14 @@ def _submit_talk(client, conference, **kwargs):
...
         'duration': conference.durations.first().id,
+        'audience_level': conference.audience_levels.first().name,
     }
 
     variables = {**defaults, **kwargs}
 
     return client.query(
         """
-        mutation($conference: ID!, $topic: ID!, $title: String!, $abstract: String!, $language: ID!, $type: ID!, $duration: ID!) {
+        mutation($conference: ID!, $topic: ID!, $title: String!, $abstract: String!, $language: ID!, $type: ID!, $duration: ID!, $audience_level: ID!) {
```
Comment (marcoacierno) [inline backend/tests/submissions/test_send_submission.py]: As it's a variable in GraphQL I would name it audienceLevel (to be consistent)

**Comment by marcoacierno** [comment]
Once we merge the strawberry migration I will rebase this PR and convert the changes here to strawberry

File: backend/tests/conferences/factories.py (hunk 1 above)
Comment (estyxx) [reply backend/tests/conferences/factories.py]: Now that we have pre-commit it should fix... no?

**Pull request closed without merging**